app/views/hyrax/file_sets/_schema_version.html.erb

```diff
@@ -0,0 +1,35 @@
+<% updating = (f.object.schema_version.to_f < Hyrax::FlexibleSchema.last.id.to_f) ? 'updating' : '' %>
+<div class='pull-right'>
+  <% if f.object.schema_version.present? %>
+    <div class='schema-version text-center'>
+      <span>Created On Version</span>
+      <h4><%= f.object.schema_version.to_f %></h4>
+    </div>
+  <% end %>
+  <div class='schema-version text-center <%= updating %>'>
+    <span>Version After Save</span>
+    <h4><%= Hyrax::FlexibleSchema.last.id.to_f %></h4>
```

Comment:
I am adding a method to Hyrax::FlexibleSchema... method current_schema_id orders by created date and then does last.id on it. We should prob use that method.

Reply:
sounds good to me